src/Three.js

```diff
@@ -156,5 +156,8 @@ export { Curve } from './extras/core/Curve.js';
 export { ShapeUtils } from './extras/ShapeUtils.js';
 export { SceneUtils } from './extras/SceneUtils.js';
 export { WebGLUtils } from './renderers/webgl/WebGLUtils.js';
+export { WEBGL_CONSTANTS } from './renderers/webgl/WebGLConstants.js';
```

Comment:
If that's the case, maybe rename the export to `constants` so you could use it as:
```js
THREE.WebGLConstants.constants.POINTS
THREE.WebGLConstants.fromGL(gl.POINTS)
THREE.WebGLConstants.toGL(THREE.FrontSide)
```